### PR TITLE
Travel blog - setup and controllers

### PR DESCRIPTION
**Pull request by jssol**
## Project requirements

-  Created a new app. Gave it a meaningful name.
-  Initialized the project with Git.
-  **Important:** do NOT use `rails generate scaffold` command. The point of this project is to show you how to set up controllers by yourself.
-  The project has Postgres database set up.
-  The app handles the URLs used in the wireframes:
-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.
-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user
-  Make sure that you added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  get "users", to: "users#index"
+  get "users/:user_id/posts/:post_id", to: "users#show_user_post"
+  get "users/:user_id/posts", to: "users#show_user_posts"
+  get "users/:user_id", to: "users#show_user"
```
Comment (ahmetbozaci) [inline config/routes.rb]: ### Optional
You can also write to routes like that.
```

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show_user; end
+
+  def show_user_posts; end
+
+  def show_user_post; end
```
Comment (ahmetbozaci) [inline app/controllers/users_controller.rb]: - [ ] For the best practice create post controller methods in another file.

```ruby
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @jssol  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Database adapter is using PostgreSQL.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: config/routes.rb (hunk 1 above)
Comment (jssol) [reply config/routes.rb]: Thanks for the feedback, I was unsure about this. I thought we should write down all the routes individually.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (jssol) [reply app/controllers/users_controller.rb]: Fantastic :100:
I am going to do that right away!

File: config/routes.rb (hunk 1 above)
Comment (ahmetbozaci) [reply config/routes.rb]: @jssol It is an easy and clean way to create routes. You can check it with run `rails routes` in the console output will be the same. But still, this is optional.

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 



@jssol  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

- Well done with changing codes according to review.
___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**